### PR TITLE
incubator-kie-issues#563: Add apache license headers

### LINKED ISSUE #563: replace copyrights headers/comments in `kogito-images`

### PR DESCRIPTION
**Pull request by jomarko**
Closes: https://github.com/apache/incubator-kie-issues/issues/563
- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by LightGuard — APPROVED** [review]

**Comment by LightGuard** [comment]
This LGTM, feel free to merge, unless we're concerned about CI failing

**Review by yesamer — CHANGES_REQUESTED** [review]
Checking CI Failures, please do not merge it.

/cc @jstastny-cz @jomarko

**Comment by domhanak** [comment]
@jomarko please rebase

**Comment by jomarko** [comment]
@domhanak thank you for a comment. Rebased.

**Review by jstastny-cz — CHANGES_REQUESTED** [review]
No changes in scriptss should appear other than license headers. Please check. Not all kiegroup URLs are being replaced.

File: .ci/jenkins/Jenkinsfile.setup-branch [hunk 1]
```diff
@@ -50,7 +50,7 @@ pipeline {
                 script {
                     dir(getRepoName()) {
                         sh '''
-                            curl -Ls https://github.com/kiegroup/kie-cloud-tools/releases/download/v1.3.6/cekit-image-validator-runner.tgz --output cekit-image-validator-runner.tgz
+                            curl -Ls https://github.com/apache/incubator-kie-kie-cloud-tools/releases/download/v1.3.6/cekit-image-validator-runner.tgz --output cekit-image-validator-runner.tgz
```
Comment (jstastny-cz) [inline .ci/jenkins/Jenkinsfile.setup-branch]: Shouldn't this one stay?

Comment (yesamer) [reply .ci/jenkins/Jenkinsfile.setup-branch]: @jomarko This repo hasn't been moved to apache, please revert this line

Comment (jomarko) [reply .ci/jenkins/Jenkinsfile.setup-branch]: thank you both for spotting

**Comment by kie-ci3** [comment]
**PR job** `#30` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/30/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-ephemeral<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-postgresql<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-task-console<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-trusty-postgresql<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-swf-builder<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-trusty-ui<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-s2i-builder<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/30/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#29` was: **ABORTED**
Possible explanation: Most probably a timeout, please review

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/29/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>Failed in branch kogito-base-builder<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-ephemeral<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-infinispan<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-mongodb<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-runtime-jvm<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-postgresql<br/>Click here to forcibly terminate running steps<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-oracle<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-mongodb<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-swf-builder<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jobs-service-ephemeral<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/29/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

File: scripts/run-bats.sh [hunk 2]
```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
+#
```
Comment (yesamer) [inline scripts/run-bats.sh:2]: @jomarko A commit modified the permissions of this file, please note:

Screenshot 2024-01-11 at 16 38 52

This affects every file marked with that label, please revert all the permissions of those files (that explains why the run-bats build is failing)

**Review by yesamer — CHANGES_REQUESTED** [review]

File: .ci/jenkins/dsl/test.sh [hunk 3]
```diff
@@ -1,4 +1,23 @@
 #!/bin/bash -e
+#
```
Comment (yesamer) [inline .ci/jenkins/dsl/test.sh:2]: @jomarko same comment for this script file

File: modules/kogito-custom-truststore/added/configure-custom-truststore.sh [hunk 4]
```diff
@@ -1,4 +1,22 @@
 #!/bin/sh
+#
```
Comment (yesamer) [inline modules/kogito-custom-truststore/added/configure-custom-truststore.sh:2]: @jomarko same comment for this script file

File: scripts/run-bats.sh (hunk 2 above)
Comment (yesamer) [reply scripts/run-bats.sh:2]: It seems to me all the *.sh files are affected :(

**Comment by kie-ci3** [comment]
**PR job** `#31` was: **ABORTED**
Possible explanation: Most probably a timeout, please review

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/31/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>Failed in branch kogito-runtime-jvm<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-infinispan<br/>Click here to forcibly terminate running steps<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-trusty-ui<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-jit-runner<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-trusty-postgresql<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-base-builder<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-data-index-ephemeral<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-swf-devmode<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-trusty-redis<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/31/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#33` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/33/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/33/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#32` was: **ABORTED**
Possible explanation: Most probably a timeout, please review

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/32/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-jobs-service-ephemeral seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-data-index-ephemeral seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/32/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by yesamer** [comment]
`WARNING: Tests on kogito-data-index-ephemeral seems to have failed` ?_?

**Comment by kie-ci3** [comment]
**PR job** `#34` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/34/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-data-index-infinispan seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>Aborted by Yeser Amer<br/>Sending interrupt signal to process<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/34/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#37` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/37/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-jobs-service-postgresql seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/37/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#39` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/39/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-s2i-builder<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/39/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Review by yesamer — APPROVED** [review]
Thank you @jomarko, just need to wait for a green build.

**Comment by yesamer** [comment]
@LightGuard Can you please review it?

**Comment by LightGuard** [comment]
I'll take a look tomorrow. :)

**Comment by kie-ci3** [comment]
**PR job** `#42` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/42/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-swf-devmode<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/42/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by kie-ci3** [comment]
**PR job** `#45` was: **FAILURE**
Possible explanation: Pipeline failure or project build failure

Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/45/display/redirect
See console log:
<details>
<summary><b>Console Logs</b></summary>
[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] unstable<br/>WARNING: Tests on kogito-jobs-service-ephemeral seems to have failed<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>[Pipeline] error<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] }<br/>Failed in branch kogito-s2i-builder<br/>[Pipeline] // parallel<br/>[Pipeline] }<br/>[Pipeline] // dir<br/>[Pipeline] }<br/>[Pipeline] // script<br/>[Pipeline] }<br/>[Pipeline] // stage<br/>[Pipeline] stage<br/>[Pipeline] { (Declarative: Post Actions)<br/>[Pipeline] script<br/>[Pipeline] {<br/>[Pipeline] sh<br/>+ wget --no-check-certificate -qO - 'https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest/job/kogito-images.build-and-test/45/api/json?depth=0'<br/>[Pipeline] readJSON<br/>[Pipeline] sh<br/>
</details>

**Comment by yesamer** [comment]
@jomarko @rodrigonull WE FINALLY GOT A GREEN CHECK!! 
Waiting for @LightGuard 's review to merge it.

**Comment by LightGuard** [comment]
LGTM! Go ahead

**Pull request merged**